ignite/services/chain/generate.go

```diff
@@ -14,18 +14,22 @@ import (
 )
 
 const (
-	defaultVuexPath    = "vue/src/store"
-	defaultDartPath    = "flutter/lib"
-	defaultOpenAPIPath = "docs/static/openapi.yml"
+	defaultVuexPath        = "vue/src/store"
+	defaultComposablesPath = "vue/src/composables"
+	defaultHooksPath       = "react/src/hooks"
+	defaultDartPath        = "flutter/lib"
+	defaultOpenAPIPath     = "docs/static/openapi.yml"
```

Comment:
We should make these paths public and move them to `chainconfig` for example, in case we want to use these defaults somewhere else. We will need to reference them when users opt-in to generate hooks or composables to be able to update their config with any of these defaults.

There is a comment down the line on `ignite/templates/app/stargate/config.yml` that is related to this change.

Reply:
done :)